server/api.go

```diff
@@ -286,6 +287,33 @@ func (p *Plugin) handleDebug(w http.ResponseWriter, r *http.Request) {
 	res.code = http.StatusNotFound
 }
 
+// handleConfig returns the client configuration, and cloud license information
+// that isn't exposed to clients yet on the webapp
+func (p *Plugin) handleConfig(w http.ResponseWriter) error {
+	license := p.pluginAPI.System.GetLicense()
+	if license == nil {
+		p.handleErrorWithCode(w, http.StatusBadRequest, "no license",
+			errors.New("no license found"))
+		return nil
+	}
```

Comment:
Surely we'd like to return the config to poor unlicensed servers as well right? :p

Reply:
haha, oops. Defaulting to `starter` seems like the best option.